ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>Refer to https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/ai-platform-unified/notebooks/notebook_template.ipynb for how variables should be named and prepared for replacement.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>